x-pack/plugins/gis/public/shared/layers/sources/ems_file_source.js

```diff
@@ -70,17 +68,12 @@ export class EMSFileSource extends AbstractVectorSource {
     };
   }
 
-  renderDetails() {
-    const emsHotLink = emsServiceSettings.getEMSHotLink(this._descriptor.id);
-    return (
-      <EuiText color="subdued" size="s">
-        <p className="gisLayerDetails">
-          <strong className="gisLayerDetails__label">Source </strong><span>Elastic Maps Service</span><br/>
```

Comment:
Can you not use the same formatting as I had added here in the new location?

Reply:
yes. Same components just rendered with different data.